client/python/pylintrc

```diff
@@ -0,0 +1,4 @@
+[MASTER]
```

Comment:
If we want to run pylint, we need to run it in CI. I am -1 to adding this file just for individual developer use.

Reply:
I have no objection whatsoever if you want to push a GHA change to add pylint (look at `.github/workflows/python.yml`) -- just make sure that it's configured to work with `black` instead of against it :)

Reply:
I think a better solution for now is me just adding these changes to my personal pylint settings. I might change my mind later ;))